platforms/software/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishSnapshotIntegTest.groovy

```diff
@@ -195,4 +195,46 @@ class MavenPublishSnapshotIntegTest extends AbstractMavenPublishIntegTest {
             snapshotVersions == ["1.0-SNAPSHOT"]
         }
     }
+
+    def "published metadata after artifacts"() {
+        settingsFile << 'rootProject.name = "snapshotPublish"'
+        buildFile << """
+    apply plugin: 'java'
```

Comment:
I understand the other tests in this file use the old plugin application syntax, but can we use the plugins block syntax for new tests?

Reply:
Looks like it works for this test indeed. I think to do that at a large scale, we will need to review some fixtures to resolve ordering issues.